### PR TITLE
chore(CollapsibleChecklist): PDTVT-1090/ISSUE-1038 Make it use the papirka check…

### PR DESCRIPTION
**Pull request by jamiek-acl**
Make the CollapsibleChecklist use the paprika checkbox
https://github.com/acl-services/paprika/issues/1038

This is needed for: https://aclgrc.atlassian.net/browse/PDTVT-1090

Before:
Screen Shot 2021-05-21 at 10 39 39 AM

After:
Screen Shot 2021-05-21 at 10 39 34 AM

### CODE REVIEW

**Review by allison-c — DISMISSED** [review]

File: packages/CollapsibleChecklists/src/components/Group/Group.js [hunk 1]
```diff
@@ -132,18 +124,16 @@ function Group(props) {
...
-          ref={checkboxRef}
-          checked={allChildItemsAreChecked}
-          type="checkbox"
-          disabled={isDisabled}
-          onChange={expandGroupAndToggleChildren}
-        />
-        {title}
-      </label>
-    </React.Fragment>
+    <label>
```
Comment (mikrotron) [inline packages/CollapsibleChecklists/src/components/Group/Group.js:127]: The `<Checkbox>` already renders a `<label>`:
https://github.com/acl-services/paprika/blob/master/packages/Checkbox/src/Checkbox.js#L49

File: packages/CollapsibleChecklists/src/components/Item/Item.js [hunk 2]
```diff
@@ -25,7 +26,14 @@ const Item = React.forwardRef((props, ref) => {
   return (
     <div css={itemStyles} isDisabled={isDisabled}>
       <label>
```
Comment (mikrotron) [inline packages/CollapsibleChecklists/src/components/Item/Item.js:28]: Same as above

File: packages/CollapsibleChecklists/src/components/Group/Group.js (hunk 1 above)
Comment (jamiek-acl) [reply packages/CollapsibleChecklists/src/components/Group/Group.js]: Ooh nice catch

**Review by mikrotron — APPROVED** [review]
llgtm 85

**Pull request merged**